main/mariadb/APKBUILD

```diff
@@ -28,49 +29,13 @@ source="https://downloads.mariadb.org/interstitial/mariadb-$pkgver/source/mariad
 	$pkgname.initd
 	fix-mysql-install-db-path.patch
 	fix-ucontext-check.patch
-	ppc-remove-glibc-dep.patch
 	pcre.cmake.patch
+	my-medium.cnf
 	"
-# dbug test fails under rootbld
-#options="!check"
-builddir="$srcdir/$pkgname-$pkgver"
-
-# secfixes:
-#   10.2.15-r0:
-#     - CVE-2018-2786
-#     - CVE-2018-2759
-#     - CVE-2018-2777
-#     - CVE-2018-2810
-#     - CVE-2018-2782
-#     - CVE-2018-2784
-#     - CVE-2018-2787
-#     - CVE-2018-2766
-#     - CVE-2018-2755
-#     - CVE-2018-2819
-#     - CVE-2018-2817
-#     - CVE-2018-2761
-#     - CVE-2018-2781
-#     - CVE-2018-2771
-#     - CVE-2018-2813 
-#   10.1.22-r0:
-#     - CVE-2017-3313
-#     - CVE-2017-3302
-#   10.1.21-r0:
-#     - CVE-2016-6664
-#     - CVE-2017-3238
-#     - CVE-2017-3243
-#     - CVE-2017-3244
-#     - CVE-2017-3257
-#     - CVE-2017-3258
-#     - CVE-2017-3265
-#     - CVE-2017-3291
-#     - CVE-2017-3312
-#     - CVE-2017-3317
-#     - CVE-2017-3318
```

Comment:
Never ever remove `secfixes`!

Reply:
@jirutka I have had maintainer to ask me to do both.

Where can I ready a "style guide" that the AL team has consensus about?

Reply:
Maintainer of the mariadb aport is @ncopa. Do I understand you correctly that @ncopa told you to remove all `secfixes`? These are not just comments, but source for the security DB.